pages/projects/virtual-concierge/index.js

```diff
@@ -0,0 +1,129 @@
+import Head from "next/head";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import { useTranslation } from "next-i18next";
+import { Layout } from "../../../components/organisms/Layout";
+import { useRouter } from "next/router";
+import { VirtualConcierge } from "../../../components/organisms/VirtualConcierge";
+import { CallToAction } from "../../../components/molecules/CallToAction";
+
+export default function Home(props) {
+  const { t } = useTranslation(["common", "vc"]);
+  const { asPath } = useRouter();
+
+  return (
+    <Layout
+      locale={props.locale}
+      langUrl={asPath}
+      // TODO not sure what the path is here
+      breadcrumbItems={[{ text: "Service Canada Labs", link: "/" }]}
```

Comment:
The `link` should redirect to the `/home`.

Reply:
Fixed